trpfancy_fields.module

```diff
@@ -26,3 +27,55 @@ function trpfancy_fields_add_field_terms() {
     tripal_insert_cvterm($term);
   }
 }
+
+
+/**
+ * Implements hook_menu().
+ */
+function trpfancy_fields_menu() {
+  // ADMISTRATIVE PAGE:
```

Comment:
This menu item needs the parent page (`admin/tripal/extension/trpfancy_fields`) defined as well. Currently the configuration page breaks up the typical flow of the extension pages.
<img width="727" alt="Screen Shot 2019-04-05 at 9 25 48 AM" src="https://user-images.githubusercontent.com/1566301/55638614-d86f8e80-5784-11e9-97ab-754724c602a4.png">

There is a way to do this without creating the page yourself. In this case it just makes a listing of all sub-pages automatically. However, I can't remember an example off the top of my head.